Add Heltec HTCC-AB01 documentation (#1363)
TheThingsIndustries/lorawan-stack-docs: doc/content/devices/models/heltec-ab01/_index.md

- **LoRaWAN version:** LoRaWAN Specification 1.0.2
- **Regional parameters version:** RP001 Regional Parameters 1.0.2 revision B

After configuring your device, select the **Register end device** button.

{{< figure src="manual-register.png" alt="Settings for manual registration" >}}

## Configuring the code

Our device is registered with {{% tts %}}, but it still needs the correct info in order to connect.

Go back to your code, and at the top you should see these lines:
```cpp
/* OTAA para*/
uint8_t devEui[] = { 0x22, 0x32, 0x33, 0x00, 0x00, 0x88, 0x88, 0x02 };
uint8_t appEui[] = { 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00 };
uint8_t appKey[] = { 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x88, 0x66, 0x01 };
```

In {{% tts %}}, under **Activation Information**, click the **eye** icon next to **AppKey** and then selecting the **< >** icon. Then copy the AppKey by clicking the **Copy to clipboard** button.

> Then replace the string between the brackets with the AppKey you just copied.

Reviewer: ```suggestion
Replace the AppKey string between the brackets in your code with the AppKey copied from {{% tts %}}.
```
